packages/l10n/src/create-l10n-injector/create-l10n-injector.tsx

```diff
@@ -2,14 +2,23 @@ import React from 'react';
 import { reportErrorToSentry } from '@commercetools-frontend/sentry';
 import { getDisplayName } from '../utils';
 
+interface Props {
```

Comment:
Why interface and not `type`? To my knowledge interfaces are a little more powerful. They can be extended and things. I'd kind of use `type` for props and `interface` for objects. At least we could use `type` for props first and then make it an `interface` when it needs extending.

Reply:
I know I am repeating myself, but I used interface because we used it in the sentry package.

https://github.com/commercetools/merchant-center-application-kit/blob/91ace37d5d7ac9334fc3a36b461d1691970f9d68/packages/sentry/src/sentry-user-tracker/sentry-user-tracker.tsx#L4

Remember,  once we merge a PR, people (me) will copy paste 😄

Reply:
Yeah. I missed it there. Sorry.

Reply:
I'm happy to change this to Props.

Reply:
The compiler suggested to prefer using interface

Reply:
https://palantir.github.io/tslint/rules/interface-over-type-literal/

Reply:
palantir 🤢

Reply:
Hm. I read otherwise. I guess it's about being restrictive where we can and allow extending only where needed to restrict the "life" of the type system of the library. 🤷‍♂

Reply:
I would also prefer to use types over interfaces but I don't know what is the convention in TS though, I always see using interfaces...also in e.g. generated code (prisma, etc) they all use interfaces.

I don't know what would be the better way to go for us...

Reply:
I found this article that describes well the differences: https://link.medium.com/GW8RdH3AlX

It also recommends to use types for props etc, and most importantly to be consistent with the usage of them.
Interfaces are mostly useful to allow consumers of the library to extend the exported types.

Reply:
Yes that‘s also how I understood it. Being restrictive with types not being extensible as long as they‘re exported and have to be extended for consumers. Thanks for the reference.